analyses/molecular-subtyping-ATRT/00-subset-files-for-ATRT.R

```diff
@@ -81,26 +81,31 @@ tmb_df <-
                               "data",
                               "pbta-snv-consensus-mutation-tmb-all.tsv"))
 
-## TODO: Read in the SV data/GISTIC output to evaluate the chr22q loss
-#        associated with SMARB1 deletions
+# Read in GISTIC `broad_values_by_arm.txt` file
+gistic_df <-
+  data.table::fread(unzip(
+    file.path(root_dir, "data", "pbta-cnv-cnvkit-gistic.zip"),
+    files = file.path(
+      "2019-12-10-gistic-results-cnvkit",
+      "broad_values_by_arm.txt"
+    ),
+    exdir = file.path(root_dir, "scratch")
+  ))
 
 #### Filter metadata -----------------------------------------------------------
 
-# Filter metadata for `ATRT` and define `location_summary` based on values in
-# `primary_site`
 atrt_df <- metadata %>%
   dplyr::filter(short_histology == "ATRT",
-                experimental_strategy == "RNA-Seq")
-
-# Write to file
-readr::write_tsv(atrt_df, file.path(results_dir, "atrt_histologies.tsv"))
+                sample_type == "Tumor",
```

Comment:
Filtering by `experimental_strategy == "RNA-Seq"` also effectively drops normal samples, but it also made it such that any ATRT samples missing RNA-seq data got dropped. We don't need to write the ATRT subset histologies file as a TSV because the CI histologies file is always the same file as a data release: 

https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/2cbf526e24d92c11f093b33aa31663546b3fbb57/analyses/create-subset-files/create_subset_files.sh#L41

The reason why we can include the full `pbta-histologies.tsv` for testing is two-fold: 1) it's pretty small and 2) having "extra" biospecimens/samples/participants in the histologies file helps test for brittle code.